src/components/Header/Header.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import './Header.scss';
-
+import tomatilloIcon from '../../greenTomato.png'
```

Comment:
Love love love how this looks! I was looking for icons as well and this is perf!